<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tester - okno główne</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelSequence">
        <property name="text">
         <string>Wybór sekwencji:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QComboBox" name="comboBoxSequence">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Pierwszy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drugi</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonAddSequence">
        <property name="text">
         <string>Dodaj sekwencję</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonDeleteSequence">
        <property name="text">
         <string>Usuń sekwencję</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonEditSequence">
        <property name="text">
         <string>Edytuj sekwencję</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" rowspan="2">
       <widget class="QPushButton" name="pushButtonConnectionSettings">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>9999999</height>
         </size>
        </property>
        <property name="text">
         <string>Konfiguracja
połączeń</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacerLogout">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>711</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonLogout">
      <property name="text">
       <string>Wyloguj</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabScenario">
       <attribute name="title">
        <string>Scenariusz</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4"/>
      </widget>
      <widget class="QWidget" name="tabTestCase">
       <attribute name="title">
        <string>Przypadki testowe</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
      <widget class="QWidget" name="tabAction">
       <attribute name="title">
        <string>Czyności</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
      <widget class="QWidget" name="tabReport">
       <attribute name="title">
        <string>Raporty</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
